config/initializers/timezone.rb

```diff
@@ -0,0 +1,2 @@
+# Set the timezone for the JavascriptAgent (libv8 only relies on the TZ variable)
+ENV['TZ'] = Time.zone.tzinfo.canonical_identifier
```

Comment:
I guess? I know many systems also set TZ, so it might be nice to use if
present? I'm not sure.
On Wed, Mar 16, 2016 at 1:29 AM Dominik Sander <notifications@github.com>
wrote:

> In config/initializers/timezone.rb
> <https://github.com/cantino/huginn/pull/1356#discussion_r56296774>:
>
> > @@ -0,0 +1,2 @@
> > +# Set the timezone for the JavascriptAgent (libv8 only relies on the TZ variable)
> > +ENV['TZ'] = Time.zone.tzinfo.canonical_identifier
>
> Not sure, shouldn't the TIMEZONE .env setting be the only source for the
> timezone Huginn uses?
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/cantino/huginn/pull/1356/files/c48ff63e7a96260540f9399201cbd7a23d65c9fa#r56296774>
>